### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by twegestal**
testie

### CODE REVIEW

File: apps/server/src/index.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ const port = 443;
 
 const server = createServer();
 
+console.log('test') //REMOVE THIS
```
Comment (LektorVinberg) [inline apps/server/src/index.js:9]: could this say "dede megadodo" instead plz?

**Review by LektorVinberg — CHANGES_REQUESTED** [review]
Could you change the print to "DeDe MegaDooodooo"?

File: apps/server/src/index.js (hunk 1 above)
Comment (twegestal) [reply apps/server/src/index.js]: har ändrat nu

Comment (LektorVinberg) [reply apps/server/src/index.js]: ok, bra jobbat

**Review by LektorVinberg — APPROVED** [review]
ser bra ut

**Pull request merged**